cmake/Modules/opm-core-prereqs.cmake

```diff
@@ -37,6 +37,7 @@ set (opm-core_DEPS
 	# DUNE dependency
 	"dune-common"
 	"dune-istl"
+	"opm-common REQUIRED"
```

Comment:
Still doesn't work for me, can you help me by checking that all relevant changes were pushed and that it works at your end?